src/PokeDetail.js

```diff
@@ -0,0 +1,38 @@
+import { Component } from 'react';
+import './PokeDetail.css'
+class PokeDetail extends Component {
+    state = { data: {} };
+
+    loadData = async () => {
+        const { id } = this.props.match.params;
+        const url = `https://pokedex-alchemy.herokuapp.com/api/pokedex/${id}`;
+        const resp = await fetch(url);
+        const data = await resp.json();
+        this.setState({ data });
+    };
+
+    componentDidMount() {
+        this.loadData();
+    }
+    
+
+    render() {
+        // const { id } = this.props.match.params;
+        const { data } = this.state;
+        return (
+            <section>
```

Comment:
Looks like you forgot to add a loading spinner to display while the data is pulling from the API.